services/apiService.ts

```diff
@@ -293,7 +293,9 @@ export const getQuestActivityData = async (id: number) => {
   }
 };
 
-export const getQuestsParticipation = async (id: number) => {
+export const getQuestsParticipation = async (
+  id: number
+): Promise<object[] | undefined> => {
```

Comment:
you need to add the type in backTypes.ts like - 
type QuestParticipants = {
  name: String;
  desc: String;
participants: String;
}[];

and import that type here as Promise<QuestParticipants>

Reply:
Alright